### PR TITLE
[IMP]10699 add module product_manager

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: product_manager/README.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Build ACL for products
+======================
+
+This module extends the functionality of ... to support ...
```
Comment (siyuancn) [inline product_manager/README.rst]: please add the description

File: product_manager/__init__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
```
Comment (siyuancn) [inline product_manager/__init__.py]: wrong copyright

File: product_manager/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Product Manager",
+    "version": "8.0.1.0.1",
+    "depends": [
+        "base",
```
Comment (siyuancn) [inline product_manager/__openerp__.py]: no need depends on base. It was depend on by other modules like product.

File: product_manager/data/res_group_data.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+        <record id="data_product_manager" model="res.groups">
+            <field name="name">Product Manager111</field>
```
Comment (siyuancn) [inline product_manager/data/res_group_data.xml]: Why the name contains '111' ?

**Comment by siyuancn** [comment]
What's the usage for the " product_manager/static/description/icon.png" ?

**Comment by Rona111** [comment]
@siyuancn  The usage for the " product_manager/static/description/icon.png is for the module logo.

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn

**Comment by siyuancn** [comment]
@Rona111 I didn't find the file which use the icon.png.

**Comment by Rona111** [comment]
@siyuancn This logo will be automatically loaded by the odoo system under this path.
@elicoidal Do we need add our company logo for other module in the future?

File: product_manager/README.rst [hunk 5]
```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Build ACL for products
+======================
+
+This module extends the functionality of Group to allow you to add a group of product manager.
```
Comment (elicoidal) [inline product_manager/README.rst:9]: There is already a group product manager in Odoo

**Comment by elicoidal** [comment]
@Rona111 I think it could be a good idea to improve the README to add logo + website in the description at the end
eg:
More information: https://www.elico-corp.com

**Comment by elicoidal** [comment]
Other than that, I would cancel this PR (low value and confusing with existing ACL)

**Comment by elicoidal** [comment]
I am not sure about the logo (heavy to maintain)

**Pull request closed without merging**